tests/page/locator-convenience.spec.ts

```diff
@@ -144,6 +144,15 @@ it('isChecked should work', async ({ page }) => {
   expect(error.message).toContain('Not a checkbox or radio button');
 });
 
+it('isChecked should work for indeterminate input', async ({ page }) => {
+  it.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/20190' });
+  await page.setContent(`<input type="checkbox" checked >`);
+  expect(await page.locator('input').isChecked()).toBe(true);
```

Comment:
Are we documenting that isChecked returns false for mixed?

Reply:
Done.